plugins/plugin-maven/che-plugin-maven-ide/src/main/resources/org/eclipse/che/plugin/maven/client/MavenLocalizationConstant.properties

```diff
@@ -39,3 +39,7 @@ window.loader.title=Resolving dependencies
 ##### Wizard Maven Page #####
 maven.page.errorDialog.title=Not valid Maven project
 maven.page.estimate.errorMessage=Source code not matches Maven project type requirements
+
+##### Preferences page #####
+maven.preferences.title=Maven
+maven.preferences.show.artifact.id.checkbox.text=Show artifact id in project explorer (in project''s root item)
```

Comment:
`(in project''s root item)` is odd, because artifact near presentable text may be present in sub modules, not only for the root project.